app/src/main/java/org/systers/mentorship/view/activities/SignUpActivity.kt

```diff
@@ -2,6 +2,8 @@ package org.systers.mentorship.view.activities
 
 import android.content.Intent
 import android.os.Bundle
+import android.view.inputmethod.InputMethodManager
+import com.google.android.material.snackbar.Snackbar
```

Comment:
This import is repeated, causing TravisCI build to fail (https://travis-ci.org/systers/mentorship-android/builds/566601607)
Can you please remove this line?